deftree/svcparse/lexer.go

```diff
@@ -68,7 +68,7 @@ func NewTokenGroup(scn *SvcScanner) *TokenGroup {
 					}
 					twostr := string(two)
 					if len(two) > 1 && two[0] == '/' {
-						str += twostr
+						str += onestr + twostr
```

Comment:
I don't understand the surrounding material enough to know, but this feels like it deserves a comment.